app/views/home/index.html.haml

```diff
@@ -2,17 +2,17 @@
 
 = render "layouts/nav"
 
-- mentor_link = "https://goo.gl/forms/3rNdjlrS9u7gZX9p1".freeze
-- volunteer_link = "https://goo.gl/forms/7Z6dZmZxEOXXpBPA3".freeze
+- mentor_link = "https://goo.gl/forms/d8WiaiM7rYaLz3SQ2".freeze
+- volunteer_link = "https://goo.gl/forms/cpmRgGxLsUUjNymG2".freeze
 
 .page.split-home#home
   .split-left
     = image_tag 'scroller.png'
   .split-right
-    %h1.title BrickHack 3
+    %h1.title BrickHack 4
     - if !Rails.configuration.hackathon['event_is_over']
       %h3 Inviting students to come together on
-    %h2.subtitle February 11-12, 2017
+    %h2.subtitle January 27-28, 2018
```

Comment:
This should probably stay as BrickHack 3 info if nothing else on the page is changing (created #518). Best handled in a separate PR - this one is pretty big already